firefox-ios/firefox-ios-tests/Tests/XCUITests/CreditCardsTests.swift

```diff
@@ -0,0 +1,81 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+class CreditCardsTests: BaseTestCase {
+    let creditCardsStaticTexts = AccessibilityIdentifiers.Settings.CreditCards.self
+
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2306967
+    // SmokeTest
+    func testAccessingTheCreditCardsSection() {
+        navigator.nowAt(NewTabScreen)
+        navigator.goto(CreditCardsSettings)
+        unlockLoginsView()
+        // Autofill Credit cards section displays
+        let addCardButton = app.buttons[creditCardsStaticTexts.AutoFillCreditCard.addCard]
+        mozWaitForElementToExist(app.staticTexts[creditCardsStaticTexts.AutoFillCreditCard.autoFillCreditCards])
+        XCTAssertTrue(addCardButton.exists)
+        XCTAssertTrue(app.switches[creditCardsStaticTexts.AutoFillCreditCard.saveAutofillCards].exists)
```

Comment:
The test fails on this line. Here's the simulator screenshot at this point. 
![Simulator Screenshot - iPhone 15 - 2023-12-05 at 18 23 00](https://github.com/mozilla-mobile/firefox-ios/assets/1740517/c3c7d19f-bcb0-4c42-a06a-ae58533b65da)

Reply:
Added an improvement that should address this fail.